src/main/resources/templates/fragments/navbar.html

```diff
@@ -0,0 +1,31 @@
+<div th:fragment="navbar">
```

Comment:
I extracted this out to it's own file so that it can be reused across multiple pages. This ensures clean code as it now only needs to be updated in on place regardless of how many pages it is implemented on. See more about clean code here:  https://www.baeldung.com/java-clean-code